lib/route/link.c

```diff
@@ -1397,7 +1397,8 @@ int rtnl_link_build_get_request(int ifindex, const char *name,
 {
 	_nl_auto_nl_msg struct nl_msg *msg = NULL;
 	struct ifinfomsg ifi;
-	__u32 vf_mask = RTEXT_FILTER_VF;
+	__u32 vf_mask = RTEXT_FILTER_VF; // Shouldn't we also fill this in like
+					 // link_request_update does?
```

Comment:
Actually it looks like sending an RTM_GETLINK message of type PF_UNSPEC would not return bridge specific attributes anyway. PF_BRIDGE seems to be needed for that. So it seems that getting MST entries is only possible when a cache is built first, specifying the family AF_BRIDGE.